### PR TITLE
Update self-signed cert instructions

### PR DESCRIPTION
**Pull request by zachmargolis**
**Why**: Customer feedback

--

cc @lauraGgit

### CODE REVIEW

File: _pages/certs.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ First, you'll need to create an `openssl.conf` file, for example:
 ```
 [ req ]
 default_bits           = 2048
-distinguished_name     = req_distinguished_name
+distinguished_name     = REQ_DISTINGUISHED_NAME
```
Comment (lauraGgit) [inline _pages/certs.md]: Actually looks like this does not generally change.

File: _pages/certs.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ First, you'll need to create an `openssl.conf` file, for example:
...
 prompt                 = no
 
-[ req_distinguished_name ]
-commonName             = idp-sandbox.login.gov
+[ REQ_DISTINGUISHED_NAME ]
```
Comment (lauraGgit) [inline _pages/certs.md]: Similarly, looks like it should stay the same.

File: _pages/certs.md [hunk 3]
```diff
@@ -22,7 +22,10 @@ countryName            = US
 emailAddress           = 18f@gsa.gov
 ```
 
-You'll need to replace `req_distinguished_name` with your agency's details.
+You'll need to replace a few parts:
```
Comment (lauraGgit) [inline _pages/certs.md]: Might suggest the following:
```
You'll need to update the information listed under the `[ req_distinguished_name ]` section in the `.conf` to match your details. 

The `commonName` will need to reflect the domain that your SP will be initiating the SAML request from.
```

Comment (zachmargolis) [inline _pages/certs.md]: Ok thanks! Updated in ca95f5722e4af172c904804ab975d46868fea928, PTAL!

**Comment by lauraGgit** [comment]
LGTM! 🚀

**Pull request merged**